pager_setup.md

```diff
@@ -1,13 +1,18 @@
-# Pager Setup
+# Pager/Phone Setup
 
 That thing you have to carry around now. This section assumes you are using a cell phone and Pagerduty. Other setups welcome.
 
+## Taking Phone Calls
+
+1. Listen with your right ear. Studies have shown that this improves a "can-do" attitude.
```

Comment:
I did some cursory searching and wasn't able to find a study matching that description (just [an article](http://www.dailymail.co.uk/sciencetech/article-2326013/Which-ear-Researchers-brain-decides-listen-mobile.html) vaguely referencing a study about brain dominance & ear choice)

> There is a strong relationship between which side of the brain is dominant and which
> ear is used to listen to a phone. Left-brain thinkers tend to use their right hand to hold 
> their phone to their right ear. ... 95 per cent of people have a dominant left side of the 
> brain. 
source: http://www.dailymail.co.uk/sciencetech/article-2326013/Which-ear-Researchers-brain-decides-listen-mobile.html